cora/signal/data/generate_powerspectra.py

```diff
@@ -0,0 +1,72 @@
+import camb
+
+from cora.signal import lsscontainers
+from cora.util import cosmology
+
+
+# Cosmological params taken from Planck 2018: Planck-TT,TE,EE+lowL+lensing+BAO
+background_params = {
+    "ombh2": 0.02242,
+    "omch2": 0.11933,
+    "H0": 67.66,
+    "omk": 0.0,
+    "TCMB": 2.7255,
+    "nnu": 3.046,
+}
+
+# Common CAMB params
+base_params = {
+    # Initial power spectrum
+    "As": 2.105e-9,
+    "ns": 0.9665,
+    # CAMB params
+    "WantTransfer": True,
+    "WantCls": False,
+    # Matter PS params
+    "redshifts": (1.0,),
+    "kmax": 10.0,
```

Comment:
Why not boost this, just to be safe? `kmax=50` will work without having to change any other camb parameters

Reply:
I could do, the power spectra are all power law extrapolated outside the range of samples, so it just depends if CAMB is going to do anything meaningful in the extended range. I agree it doesn't really hurt though.

Reply:
I've bumped this all the way to k=1000.